<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addressListItem</class>
 <widget class="QWidget" name="addressListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0,0,0,0.5); color: white;</string>
  </property>
  <widget class="QLabel" name="add_username">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>8</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="add_header">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>8</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="add_statue">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>32</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
